src/scripts/createForm.js

```diff
@@ -0,0 +1,125 @@
+import { createElement } from './utils';
+import { addRow } from './tableActions';
+import { pushNotification } from './createNotification';
+
+export function createForm(table) {
+  const form = document.createElement('form');
+
+  const selectorOptions = [
+    'Tokyo',
+    'Singapore',
+    'London',
+    'New York',
+    'Edinburgh',
+    'San Francisco',
+  ];
+
+  form.append(
+    createInput('Name:', 'name', 'text'),
+    createInput('Position:', 'position', 'text'),
+    createSelector('Office', 'office', selectorOptions),
+    createInput('Age:', 'age', 'number'),
+    createInput('Salary:', 'salary', 'number'),
+    createSubmitButton(table, form, 'Save to table'),
+  );
+
+  form.classList.add('new-employee-form');
+  document.body.append(form);
+}
+
+function createSelector(labelText, selectorName, selectorOptions) {
+  const label = createElement('label', labelText);
+  const selector = document.createElement('select');
+
+  selectorOptions.forEach((optionText) => {
+    selector.append(createElement('option', optionText));
+  });
+
+  selector.name = selectorName;
+  selector.dataset.qa = selectorName;
+
+  label.append(selector);
+
+  return label;
+}
+
+function createInput(labelText, inputName, inputType = 'text') {
+  const input = document.createElement('input');
+  const label = createElement('label', labelText);
+
+  input.type = inputType;
+  input.name = inputName;
+  input.autocomplete = false;
```

Comment:
The `autocomplete` attribute should be set to a string value like 'off' or 'on'. Setting it to a boolean value `false` is incorrect and will not work as expected.